tools/ci/ci_wptrunner_infrastructure.sh

```diff
@@ -6,19 +6,24 @@ WPT_ROOT=$SCRIPT_DIR/../..
 cd $WPT_ROOT
 
 test_infrastructure() {
-    TERM=dumb ./wpt run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $1 $PRODUCT infrastructure/
+    PY3_FLAG="$2"
+    TERM=dumb ./wpt $PY3_FLAG run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $1 $PRODUCT infrastructure/
 }
 
 main() {
-    PRODUCTS=( "firefox" "chrome" )
+    if [[ $# -eq 1 && "$1" = "--py3" ]]; then
```

Comment:
Out of interest, what difference does it make if we don't check the number of arguments here? I would expect the second condition to fail in any case even if `$1` is empty.